### PR TITLE
fix: remove inconsistent +1e-5 offset in entropy loss log_softmax

### PR DESCRIPTION
**Pull request by Mr-Neutr0n**
## Summary

In `compute_entropy_loss()` in `janus/models/vq_model.py`, `probs` and `log_probs` should represent the same probability distribution, but they diverge because of an unnecessary `+ 1e-5` offset applied before `log_softmax`:

```python
probs = F.softmax(flat_affinity, dim=-1)
log_probs = F.log_softmax(flat_affinity + 1e-5, dim=-1)  # +1e-5 changes the input distribution
```

The `+ 1e-5` offset shifts the logits before computing `log_softmax`, which means `log_probs != log(probs)`. This causes the sample entropy term (`-sum(probs * log_probs)`) to not be a true entropy computation, introducing a small but systematic bias in the entropy loss.

`F.log_softmax` is already numerically stable (it uses the log-sum-exp trick internally), so the `+ 1e-5` offset is not needed for numerical safety and only serves to make `probs` and `log_probs` inconsistent.

## Fix

Remove the `+ 1e-5` so both `probs` and `log_probs` are computed from the same input:

```python
log_probs = F.log_softmax(flat_affinity, dim=-1)
```

## Test plan

- [ ] Verify that `compute_entropy_loss` produces correct entropy values by checking `log_probs == log(probs)` for sample inputs
- [ ] Confirm no numerical instability is introduced (expected: none, since `F.log_softmax` is inherently stable)
- [ ] Run existing VQ model training/evaluation to validate no regression

### CODE REVIEW

**Comment by Mr-Neutr0n** [comment]
Closing this one to tidy up my open pull requests.

It's been open around five months with no review activity, and I'd rather withdraw it than leave it sitting in your queue. Nothing needed from you, and no hard feelings at all.

If the fix is still wanted, this can be reopened, or I'm happy to redo it properly against current main. Apologies for the noise.
